packages/modals/src/modal/utils.tsx

```diff
@@ -0,0 +1,21 @@
+export interface TrackEvent {
+  event: string;
+  [key: string]: unknown;
+}
+
+type TrackFunc = (event: TrackEvent) => void;
+
+let trackFunc: TrackFunc;
+
+export const configure = (options: {
+  track?: TrackFunc
+}) => {
+  const { track } = options;
+  if (track === undefined) return;
```

Comment:
1. breaking change to how the function works
2. no way to reset `trackFunc` now
3. why?